File: Tema_1/CalculoSuma.pas
{Fichero CalculoSuma.pas}
#include externs.h
Program CalculoSuma
var
  x, y, result:real;
begin
  writeln('Introduce el primer sumando');
  readln(x);
  writeln('Introduce el segundo sumando');
  readln(y);
  result = suma(x,y);
  writeln('El resultado es: ', result);
end.

File: Tema_1/CalculoSuma.f90
!Programa principal en Fortran 90
program CalculoSuma
   implicit none
   real::x, y, result
   real, external::suma
   write(*,*) 'Introduce el primer sumando'
   read(*,*)x
   write(*,*)'Introduce el segundo sumando'
   read(*,*)y
   result = suma(x,y)
   write(*,*) 'El resultado es: ', result
end program CalculoSuma